src/Printer_CartridgeInfo.php

```diff
@@ -207,89 +187,74 @@ public static function getSpecificValueToSelect($field, $name = '', $values = ''
         return parent::getSpecificValueToSelect($field, $name, $values, $options);
     }
 
-    private static function getProgressColorsForColor($color)
+    /**
+     * Create a badge for a specific type of cartridge information
+     *
+     * @param array $data
+     * @param string $type
+     */
+    private static function createCartridgeInformationBadge($data, $type): ?string
     {
-        $fg_transparency_hex = '80';
-        switch ($color) {
-            case 'cyan':
-            case 'light_cyan':
-                return [
-                    'fg' => '#00ffff' . $fg_transparency_hex,
-                    'text' => 'inherit'
-                ];
-            case 'magenta':
-            case 'light_magenta':
-                return [
-                    'fg' => '#ff00ff' . $fg_transparency_hex,
-                    'text' => 'inherit'
-                ];
-            case 'yellow':
-                return [
-                    'fg' => '#ffff00' . $fg_transparency_hex,
-                    'text' => 'inherit'
-                ];
-            case 'black':
-            case 'grey':
-            case 'darkgrey':
-                return [
-                    'fg' => '#303030' . $fg_transparency_hex,
-                    'text' => 'inherit'
-                ];
+        $color_aliases = [
+            'magenta'   => 'purple',
+        ];
+        $color_translations = [
+            'black'         => __('Black'),
+            'cyan'          => __('Cyan'),
+            'magenta'       => __('Magenta'),
+            'yellow'        => __('Yellow'),
+        ];
+
+        if (
+            is_array($data)
+            && isset($data['property'])
+            && isset($data['value'])
+            && str_starts_with($data['property'], $type)
+        ) {
+            $color = str_replace($type, '', $data['property']);
+            $templateContent = <<<TWIG
+                <span class="badge bg-{{ color }} text-{{ color }}-fg fw-bold" aria-label="{{ color_translated }}">
```

Comment:
`aria-label` is redundant here. It would have only been needed if the color name wasn't shown at all.